src/lib/extend.js

```diff
@@ -79,10 +79,10 @@ const extend = {
     },
     light_onoff_brightness_color: (options={}) => {
         options = {
-            disableEffect: false, supportsHS: false, preferHS: false, disablePowerOnBehavior: false,
+            disableEffect: false, supportsHueAndSaturation: false, preferHS: false, disablePowerOnBehavior: false,
```

Comment:
@Koenkk I think this made it so we didn't noticed the wrong default being used in `light.readColorAttributes()`

It's certainly confusing. Can you give this PR an extra careful look as it's rather complex/confusing at times with all the layers of abstraction.